src/cascadia/TerminalControl/SearchBoxControl.xaml

```diff
@@ -0,0 +1,186 @@
+﻿<UserControl
+    x:Class="Microsoft.Terminal.TerminalControl.SearchBoxControl"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:local="using:Microsoft.Terminal.TerminalControl"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    mc:Ignorable="d"
+    x:Name="Root"
+    HorizontalAlignment="Stretch"
+    VerticalAlignment="Top"
+    d:DesignHeight="55"
+    d:DesignWidth="285"
+    TabNavigation="Cycle">
+
+    <UserControl.Resources>
+        <ResourceDictionary>
+            <Style x:Key="ToggleButtonStyle" TargetType="ToggleButton">
+                <Setter Property="Width" Value="25" />
+                <Setter Property="Height" Value="25" />
+                <Setter Property="Background" Value="Transparent" />
+                <Setter Property="Foreground" Value="{ThemeResource ToggleButtonCustomForeground}"/>
+                <Setter Property="Padding" Value="1" />
+                <Setter Property="CornerRadius" Value="2"/>
+            </Style>
+            <Style x:Key="ButtonStyle" TargetType="Button">
+                <Setter Property="Width" Value="25" />
+                <Setter Property="Height" Value="25" />
+                <Setter Property="Background" Value="Transparent" />
+                <Setter Property="CornerRadius" Value="2"/>
+            </Style>
+            <ResourceDictionary.ThemeDictionaries>
+                <ResourceDictionary x:Key="Default">
+                    <Style x:Key="FontIconStyle" TargetType="FontIcon">
+                        <Setter Property="FontSize" Value="12" />
+                    </Style>
+                    <Style x:Key="SearchBoxBackground" TargetType="StackPanel">
+                        <Setter Property="Background" Value="#333333" />
+                    </Style>
+                    <!-- TextBox colors !-->
+                    <SolidColorBrush x:Key="TextControlBackground" Color="#333333"/>
+                    <SolidColorBrush x:Key="TextControlPlaceholderForeground" Color="#B5B5B5"/>

```

Comment:
According to Kayla's design, place holder text should be #B5B5B5

Reply:
Even in dark mode? Because based on her specification, it looks like the placeholder text is _light_ in dark mode...

![](https://user-images.githubusercontent.com/48369326/70088713-ae989100-15cb-11ea-8750-58d14d1315e1.png)

Reply:
Use fixed color for place holder text now.